samples/Dantooine/Dantooine.Server/Migrations/20210201201010_Init.Designer.cs

```diff
@@ -0,0 +1,510 @@
+// <auto-generated />
```

Comment:
Migrations don't age very well. Please remove them and use the same worker-based initialization pattern used in the other samples.

Reply:
This was taken from the Velusia.Server example

Reply:
removed